smoketests/packages/node-js-cjs/index.cjs

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const { BacktraceClient } = require('@backtrace/node');
+const path = require('path');
+
+const SUBMISSION_URL = process.argv[2];
+if (!SUBMISSION_URL) {
+    throw new Error('submission URL is required');
+}
+
+console.log('initializing BacktraceClient');
+const client = BacktraceClient.initialize({
+    url: SUBMISSION_URL,
+    rateLimit: 5,
+    userAttributes: {
+        'custom-attribute': 'test',
+        'custom-annotation': {
+            prop1: true,
+            prop2: 123,
+        },
+    },
+    database: {
+        enable: true,
+        path: path.join(process.cwd(), 'database'),
+        captureNativeCrashes: true,
+        createDatabaseDirectory: true,
+    },
+});
+
+console.log('submitting message');
+client.send('test message').then((result) => {
```

Comment:
can we please send more? at least an exception + message. We can also send unhandled exception if we instruct node correctly.

Reply:
Added some new tests.